src/hotspot/share/c1/c1_Compiler.cpp

```diff
@@ -50,8 +50,8 @@ Compiler::Compiler() : AbstractCompiler(compiler_c1) {
 
 void Compiler::init_c1_runtime() {
   BufferBlob* buffer_blob = CompilerThread::current()->get_buffer_blob();
-  Runtime1::initialize(buffer_blob);
   FrameMap::initialize();
+  Runtime1::initialize(buffer_blob);
```

Comment:
Why this change?

Reply:
Ah, I guess it's becuase you refer to `FrameMap::caller_save_cpu_reg_at(i)` in your code.

Reply:
Yes.